PasswordRulesSharp/Generator/Generator.cs

```diff
@@ -10,10 +10,20 @@
 
 namespace PasswordRulesSharp.Generator
 {
+    /// <summary>
+    /// A service that produces passwords (with some randomness) that will
```

Comment:
Doesn't this produce cryptographically sound random bytes?

Reply:
It internally uses a cryptographic RNG, but only within the ultimate (concatenated) character class. It first shuffles that (non-cryptographically), then picks an element at (cryptographic) random each time.

But, for example, it does _not_ randomize the length of passwords, even if a rule allows for different lengths. So I didn't want to oversell the case by claiming they're fully random.

Come to think of it, this explanation perhaps belongs in the comment…